microsoft-edge/web-platform/whats-new/edge-web-platform-91.md

```diff
@@ -0,0 +1,69 @@
+<!--
+description: These highlights focus on notable, developer-consumable updates to the web platform. We are continually working on exciting projects, fundamental improvements, and consumer-facing platform changes and are delighted to share what we are releasing in Edge.
+title: What's New to Web Platform (Microsoft Edge 92)
+author: MSEdgeTeam
+ms.author: kanderson
+ms.date: 06/04/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, Web Platform
+-->
+
+# What's New to Web Platform (Microsoft Edge 91)  
+
+These highlights focus on notable, developer-consumable updates to the web platform. We are continually working on exciting projects, fundamental improvements, and consumer-facing platform changes and are delighted to share what we are releasing in Edge 91.  
+
+🚢 Edge 91 shipped to Stable audiences on May 27, 2021.  
+
+[Import assertions](https://github.com/tc39/proposal-import-assertions)  and [JSON modules](https://github.com/tc39/proposal-json-modules). These allow developers to import JSON data, such as localized strings, into their JavaScript environments via the same ES module dependency system for JavaScript dependencies. 
+
+
+## Site Compatibility Changes
+
+<!-- Title: Site Compatibility Changes -->  
+
+*   For standards-compliance, [a change was made for Edge 91](https://chromium.googlesource.com/chromium/src/+/b332c5609ed9caa5979c24a0d1bdea242329258c) to the handling of SameSite=Lax cookies to prevent them from being sent on cross-site subframe navigations. If this change breaks a scenario, the cookie in question will need to be marked SameSite=None. Note: Chrome shipped this change in Chrome v90 while Edge is shipping in v91. 
+
+*   See more impacting changes that are coming to Microsoft Edge, [here](https://docs.microsoft.com/en-us/microsoft-edge/web-platform/site-impacting-changes). 
```

Comment:
Global: delete "/en-us" from Microsoft URLs, to make build errors/warnings go away.

Reply:
Global: any URLs that start with 
https://docs.microsoft.com/en-us/microsoft-edge/
or
https://docs.microsoft.com/microsoft-edge/
are in the present docs repo and should be changed to relative paths, to make build errors go away.